target/linux/ramips/dts/rt5350_tenda_a6.dts

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tenda,a6", "ralink,rt5350-soc";
+	model = "Tenda A6";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "a6:blue:power";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
```

Comment:
Please try "gpio-keys" here. `poll-interval` can be removed then.

Reply:
@mans0n Is that supported for the old subtargets?

When grepping, I did not find a single case of "gpio-keys" for any rt**** devices.

Reply:
Probably. At least I've tested it on cy-swr1100.